tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ES2020",
+    "allowJs": true,
+    "checkJs": true,
```

Comment:
Oops, I think this was maybe a bad suggestion as now all the imports have errors because they're implictly any.

Reply:
yes, I just disabled it again....